scripts/gunicorn.conf

```diff
@@ -1,6 +1,6 @@
-bind = '127.0.0.1:8000'
+bind = '0.0.0.0:8000'
 backlog = 2048
-workers = 1
+workers = 4
```

Comment:
Out of curiosity: Are the hardcoded 4 workers something I should change depending on my architecture or is it a sane default?

Reply:
I took it as an “okay” default for most systems.
This [blog post](https://medium.com/building-the-system/gunicorn-3-means-of-concurrency-efbb547674b7) gives a rule of thumb of 2xCPUs + 1.
They also talk about different worker types for different workloads. I wanted to look into this next:)

Reply:
[The official documentation seems pretty good as well and has some more details on the workers and everything](http://docs.gunicorn.org/en/latest/design.html#how-many-workers). 

I guess for Paperless there is often just one client connecting to it. So I would probably be more concerned about having too many workers for a simple system than not enough.

Reply:
I removed the worker increase from this PR, separation of concerns etc. For most systems this will still make sense, so maybe we should open a separate PR for this.